File: CS/ViewerCustomCommand/CustomNextPageCommand.cs
using DevExpress.XtraPdfViewer;
using DevExpress.XtraPdfViewer.Commands;
using System.Windows.Forms;


namespace ViewerCustomCommand {
    public class CustomNextPageCommand : PdfNextPageCommand {
        public CustomNextPageCommand(PdfViewer control)
            : base(control) {
        }

        public override void Execute() {
            MessageBox.Show("Custom command executed");
            base.Execute();
        }
    }
}

File: VB/ViewerCustomCommand/CustomNextPageCommand.vb
Imports DevExpress.XtraPdfViewer
Imports DevExpress.XtraPdfViewer.Commands
Imports System.Windows.Forms

Namespace ViewerCustomCommand

    Public Class CustomNextPageCommand
        Inherits PdfNextPageCommand

        Public Sub New(ByVal control As PdfViewer)
            MyBase.New(control)
        End Sub

        Public Overrides Sub Execute()
            MessageBox.Show("Custom command executed")
            MyBase.Execute()
        End Sub
    End Class
End Namespace
